script.embuary.helper/resources/lib/helper.py

```diff
@@ -0,0 +1,206 @@
+#!/usr/bin/python
+# coding: utf8
+
+########################
+
+import xbmc
+import xbmcaddon
+import xbmcgui
+import simplejson
+
+########################
+
+ADDON = xbmcaddon.Addon()
+ADDON_ID = ADDON.getAddonInfo('id')
+
+NOTICE = xbmc.LOGNOTICE
+WARNING = xbmc.LOGWARNING
+DEBUG = xbmc.LOGDEBUG
+LOG_ENABLED = True if ADDON.getSetting('log') == 'true' else False
```

Comment:
In v18 a new function `getSettingBool(...)` was added.
```suggestion
LOG_ENABLED = True if ADDON.getSettingBool('log')
```

Reply:
Both is ok and working.
I have to use getSetting because of Krypton and getSettingBool requires Leia.